components/Event.tsx

```diff
@@ -136,54 +117,33 @@ export const Event: FC<{ event: NFTEvent }> = ({ event }) => {
       </div>
       <div className={styles.imgOverlay}>
         <h4 className={styles.nameLabel}>
-          <a
-            target="_blank"
-            rel="noreferrer"
-            href={event.assetUrl}
-            className={styles.nameLabelLink}
-          >
+          <a target="_blank" rel="noreferrer" href={event.assetUrl} className={styles.nameLabelLink}>
             {trunicate(event.assetName, 80)}
           </a>
-          <a
-            target="_blank"
-            rel="noreferrer"
-            href={event.collectionUrl}
-            className={styles.subLabelLink}
-          >
+          <a target="_blank" rel="noreferrer" href={event.collectionUrl} className={styles.subLabelLink}>
             {trunicate(event.collectionName, 50)}{" "}
           </a>
         </h4>
       </div>
       <div className={styles.eventDetails}>
         <div className={styles.titleContainer}>
-          <h3 className={`${styles.title} ${styles.titleItem}`}>
-            {getTitle(event)}
-          </h3>
+          <h3 className={`${styles.title} ${styles.titleItem}`}>{getTitle(event)}</h3>
           <div className={styles.titleItem}>
-            <AddressContainer
-              address={otherAddress}
-              shouldResolveENS={isVisible}
-            />
+            <AddressContainer address={otherAddress} shouldResolveENS={isVisible} />
           </div>
           {(event.action === "Sold" || event.action === "Bought") && (
             <h3 className={`${styles.priceDesc} ${styles.titleItem}`}>
               {"for"}
               <div className={styles.priceContainer}>
-                <img
-                  src="/ethereum_icon.svg"
-                  className={styles.ethIcon}
-                  alt="Ethereum Icon"
-                />
+                <img src="/ethereum_icon.svg" className={styles.ethIcon} alt="Ethereum Icon" />
```

Comment:
might want to take the svgs and create a const file and import them for better readability

const ETHER_SVG = '/etherium_icon.svg';

```js
<img src={ETHER_SVG} />
```

Reply:
I agree, not within the scope of this PR though